src/base/numbers.c

```diff
@@ -74,3 +74,42 @@ int _getHexString(char *buffer, unsigned long hexnumber, bool use_capital) {
     }
     return size;
 }
+
+// Unsigned
+int itoa(char *buffer, unsigned long number, int base, bool use_capital) {
```

Comment:
in numbers.h this is `_itoa` while in numbers.c is `itoa`, i would say that since `itoa` is  name for a stdlib function let's call it something else since it can convert to any base, it can be something like `getNumericString` or something similar.